DigitalLearningSolutions.Web/Helpers/MigrationHelperMethods.cs

```diff
@@ -47,7 +47,8 @@ public static IServiceCollection RegisterMigrationRunner(this IServiceCollection
                         typeof(FixCustomisationUpdateSPs).Assembly,
                         typeof(AddEmailToFrameworkCollaboratorTable).Assembly,
                         typeof(AddFrameworkReviewsTable).Assembly,
-                        typeof(AddArchivedDateToFrameworkReviews).Assembly
+                        typeof(AddArchivedDateToFrameworkReviews).Assembly,
+                        typeof(AddMapDisplayColumnToCentres).Assembly

```

Comment:
Do you know if you actually need to add this? I'd have thought all these classes would be in the same assembly, but I don't confess to understand assemblies or FluentMigrator particularly well.

Reply:
You are absolutely right. I was blindly following the readme instructions for setting up new migrations, which mentions adding each one. I've tested my new migration without the addition in ScanIn, and it still works fine (some debug comparisons show that each of these assemblies are definitely exactly the same, so we only need one as they are all from the same assembly).

@stellake  if you think we should, I am happy to cutback the ScanIn procedure to a single assembly and update the readme file as part of this ticket.

Reply:
Sounds good! 👍